### PR TITLE
Added base template for all checkers

### PR DESCRIPTION
**Pull request by DavidBarker-EE**
- Removed webpack
- Removed build sequences and replace with sass cli
- Added route handler for basic template
- Updated dockerfile with correct node version
- Added static file serving routes to HAPI

### CODE REVIEW

File: public/assets/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*
```
Comment (Andrew-Folga) [inline public/assets/.gitignore]: Why having multiple .gitignore files instead of one in root folder?

**Comment by Andrew-Folga** [comment]
LGTM just a minor comment.

File: public/assets/.gitignore (hunk 1 above)
Comment (DavidBarker-EE) [reply public/assets/.gitignore]: I've always found it easier to manage what's ignored this way, but I'm not bothered either way.

Comment (Andrew-Folga) [reply public/assets/.gitignore]: My problem is it is more difficult to maintain. Also do child files overwrite parent files? What if we miss .gitignore in one of the children - do we then control those children from parent one?

Comment (Andrew-Folga) [reply public/assets/.gitignore]: I am open to options but I've never seen it in such setup hence need to know more.

Comment (swdpcomputing) [reply public/assets/.gitignore]: Would agree with Andrew - one is sufficient imo.

Comment (DavidBarker-EE) [reply public/assets/.gitignore]: Ok, let's get it all into one file 👍

**Review by swdpcomputing — APPROVED** [review]

**Review by Andrew-Folga — APPROVED** [review]
LGTM

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,11 @@
-FROM defradigital/node-development:22.4.0-alpine3.19 as dev
+FROM defradigital/node-development:2.3.0-node22.4.0 as dev
 
 WORKDIR /usr/src/app
 COPY package*.json ./
 RUN npm ci --only=development
 COPY . .
 
-RUN npm run build:dev
```
Comment (alanplatt) [inline Dockerfile]: I think we should have a chat where we are going with this as we're diverging from https://defra.github.io/ffc-development-guide/development-patterns/containers/

Comment (DavidBarker-EE) [reply Dockerfile]: How so? we haven't got a prod container but I don't see anything wrong here.

**Pull request closed without merging**